src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -302,6 +302,19 @@ public function getJobTypeString()
         return isset($map[$this->JobType]) ? $map[$this->JobType] : '(Unknown)';
     }
 
+    /**
+     * @return string
+     */
+    public function getImplementationSummary()
```

Comment:
The use of the word 'summary' is a little weird, perhaps rename it to `getTrimmedImplementation()` or similar?.  I'd also add example to show that its trimming off the front of the namespace